### PR TITLE
Secure Cookies and Limit request header size/ form fields

### PR DESCRIPTION
**Pull request by sanjaya902**
Cookies security has been configured in the settings.py file to ensure cookies are only sent over HTTPS, prevents JavaScript from accessing session cookies, mitigate CSRF attacks by restricting cross-origin cookie sharing, ensure CSRF cookies are sent over HTTPS only, enhance CSRF protection and ensure DEBUG is set to False in production to avoid sensitive information exposure.

### CODE REVIEW

**Comment by sanjaya902** [comment]
Sorry about the mess, I did few commits for the same changes.

File: core/settings.py [hunk 1]
```diff
@@ -48,6 +48,26 @@
...
+
+#Ensure CSRF cookies are sent over HTTPS only
+CSRF_COOKIE_SECURE = True
+
+#Enhance CSRF protection
+CSRF_COOKIE_SAMESITE = 'Strict'
+
+#Ensure DEBUG is set to False in production to avoid sensitive information exposure
+DEBUG = False
+
```
Comment (s-bonnor) [inline core/settings.py:69]: Do not commit changing DEBUG to false
please update the pull request setting DEBUG back to true

**Review by s-bonnor — CHANGES_REQUESTED** [review]

File: core/settings.py (hunk 1 above)
Comment (sanjaya902) [reply core/settings.py]: Thanks for the feedback. Requested change has been updated.

**Comment by sanjaya902** [comment]
I have committed new charges to limit request header size and form fields.

**Review by s-bonnor — APPROVED** [review]
Looks good, will be merged in a little bit

**Pull request merged**